src/Powercord/plugins/pc-moduleManager/components/manage/Themes.jsx

```diff
@@ -56,44 +57,64 @@ class Themes extends Base {
-              frameBorder='0' allow='accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture'
-              allowFullScreen
-            />
-          </div>
-          <div className='big'>
-            enjoying the beta? <a href={`https://youtube.com/watch?v=${encodeIdentifier(FEEDBACK_IDENTIFIER)}`} target='_blank'>gib feedback</a>
-          </div>
-          <div className='small right'>install <a href='https://github.com/redstonekasi/theme-toggler' target='_blank'>theme-toggler</a></div>
-        </div>
-      );
-    }
+  // renderBody () {
+  //   if (this.state.tryBeta) {
+  //     return (
+  //       <div className='powercord-text beta-container'>
+  //         <div className='very-big'>welcome to the theme manager beta</div>
+  //         <div className='iframe-wrapper'>
+  //           <iframe
+  //             width='100%' height='100%'
+  //             src={`https://www.youtube.com/embed/${encodeIdentifier(BETA_IDENTIFIER)}?autoplay=1`}
+  //             frameBorder='0' allow='accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture'
+  //             allowFullScreen
+  //           />
+  //         </div>
+  //         <div className='big'>
+  //           enjoying the beta? <a href={`https://youtube.com/watch?v=${encodeIdentifier(FEEDBACK_IDENTIFIER)}`} target='_blank'>gib feedback</a>
+  //         </div>
+  //         <div className='small right'>install <a href='https://github.com/redstonekasi/theme-toggler' target='_blank'>theme-toggler</a></div>
+  //       </div>
+  //     );
+  //   }
+
+  // return (
+  //   <div className='developerPortalCtaWrapper-2PniQs'>
+  //     <div className='placeholderImage-17g3fb desaturate-_Twf3u'/>
+  //     <div className='colorStandard-21JIj7 size14-3fJ-ot developerPortalCtaText-3k7ne7'>
+  //       Theme manager beta is here! Be careful, it's still in beta and glitches may occur.
+  //     </div>
+  //     <Button className='developerPortalCta-3QrlHn' onClick={() => this.setState({ tryBeta: true })}>
+  //       Try the beta
+  //     </Button>
+  //   </div>
+  // );
 
+  //   return this.getItems().map((theme) => {
+  //     return this.renderItem(theme);
+  //   });
+  // }
+
+  renderItem (item) {
     return (
-      <div className='developerPortalCtaWrapper-2PniQs'>
-        <div className='placeholderImage-17g3fb desaturate-_Twf3u'/>
-        <div className='colorStandard-21JIj7 size14-3fJ-ot developerPortalCtaText-3k7ne7'>
-          Theme manager beta is here! Be careful, it's still in beta and glitches may occur.
-        </div>
-        <Button className='developerPortalCta-3QrlHn' onClick={() => this.setState({ tryBeta: true })}>
-          Try the beta
-        </Button>
-      </div>
+      <InstalledProduct
+        product={item.manifest}
+        isEnabled={powercord.styleManager.isEnabled(item.entityID)}
+        onToggle={async v => {
+          await this._toggle(item.entityID, v);
+          this.forceUpdate();
```

Comment:
just make this a functional component where you pass some functions and use setState ??